torch/__init__.py

```diff
@@ -823,3 +823,6 @@ def _register_device_module(device_type, module):
         raise RuntimeError("The runtime module of '{}' has already "
                            "been registered with '{}'".format(device_type, getattr(m, device_type)))
     setattr(m, device_type, module)
+
+# expose return_types
+from . import return_types
```

Comment:
I'm confused (and not very good with Python modules and imports) -- could this just be `return_types = torch._C._return_types`?

Reply:
This works in Python 3.10. But CI looks to be failing on 3.6 :(

Reply:
Surprisingly `return_types = torch._C._return_types` works locally on Python 3.6 as well (and tests pass as well) :(

```
Python 3.6.13 |Anaconda, Inc.| (default, Jun  4 2021, 14:25:59) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> import torch.return_types
```